### PR TITLE
feat: Make pipeline required

### PR DESCRIPTION
**Pull request by raulb**
https://github.com/meroxa/platform-api/pull/560

Part of https://github.com/meroxa/product/pull/128

- [x] Add tests
- [x] Re-generate documentation using `tfplugindocs`
- [x] Add changelog
- [x] Update github action

### CODE REVIEW

**Comment by raulb** [comment]
@terrancej I'm not sure what's the best way to test this and also why these acceptance tests are failing. Any guidance?

File: meroxa/resource_connector.go [hunk 1]
```diff
@@ -76,16 +76,14 @@ func resourceConnector() *schema.Resource {
 				Elem:        schema.TypeString,
 			},
 			"pipeline_id": {
-				Type:        schema.TypeInt,
-				Description: "Connector's Pipeline ID, uses default pipeline if not specified",
-				Optional:    true,
-				Computed:    true,
+				Type:         schema.TypeInt,
+				Description:  "Connector's Pipeline ID, uses default pipeline if not specified",
+				ExactlyOneOf: []string{"pipeline_name"},
```
Comment (terrancej) [inline meroxa/resource_connector.go:81]:
```suggestion
				ExactlyOneOf: []string{"pipeline_name", "pipeline_id"},
```

File: meroxa/resource_connector.go [hunk 2]
```diff
@@ -76,16 +76,14 @@ func resourceConnector() *schema.Resource {
...
 			},
 			"pipeline_name": {
-				Type:        schema.TypeString,
-				Description: "Connector's Pipeline Name, uses default pipeline if not specified",
-				Optional:    true,
-				Computed:    true,
+				Type:         schema.TypeString,
+				Description:  "Connector's Pipeline Name, uses default pipeline if not specified",
+				ExactlyOneOf: []string{"pipeline_id"},
```
Comment (terrancej) [inline meroxa/resource_connector.go:86]:
```suggestion
```

**Review by terrancej — CHANGES_REQUESTED** [review]
exactly oneof needs to name of the keys on only of the keys I put in suggestions. Add a test that adds both ensure it fails, add a test that adds none ensure it fails and then maybe a test for both individually

**Review by terrancej — APPROVED** [review]
LGTM
